pep-0723.rst

```diff
@@ -92,25 +92,38 @@ The following are some of the use cases that this PEP wishes to support:
   continue working without having to learn a new format. More likely, even, is
   that tooling will eventually support this transformation with a single
   command.
-* Tooling that wishes to modify metadata on behalf of the user, for example,
-  package managers or dependency update automation in CI.
+* Users that wish to avoid manual dependency management. For example, package
+  managers that have commands to add/remove dependencies or dependency update
+  automation in CI that triggers based on new versions or in response to
+  CVEs [1]_.
 
 
 Specification
 =============
 
 Any Python script may assign a variable named ``__pyproject__`` to a multi-line
 *double-quoted* string literal (``"""``) containing a valid TOML document. The
-opening of the string MUST be on the same line as the assignment. The closing
-of the string MUST be on a line by itself, and MUST NOT be indented.
+variable MUST start at the beginning of the line and the opening of the string
+MUST be on the same line as the assignment. The closing of the string MUST be
+on a line by itself, and MUST NOT be indented.
 
-When there are multiple ``__pyproject__`` variables defined, tools MUST
-produce an error.
+When there are multiple ``__pyproject__`` variables defined in this way, tools
```

Comment:
This reads like a loosening -- I would say tools should error if there are multiple assignments to ``__pyproject__``, regardless of validity.

```suggestion
When there are multiple ``__pyproject__`` variables defined, tools
```

Reply:
Done!